src/main/java/Duke/Tasks/Task.java

```diff
@@ -0,0 +1,44 @@
+package Duke.Tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+public class Task {
+    private Boolean done = false;
```

Comment:
Will using isDone rather than done make the name some more boolean.

Reply:
Noted and I tried to make some of the variable names more intuitive or added comments to clarify